__tests__/index.js

```diff
@@ -1,14 +1,25 @@
 const moment = require("moment");
 const date = require("date-fns");
 const fr = require("date-fns/locale/fr");
+const dayjs = require("dayjs");
+const RelativeTime = require("dayjs/plugin/RelativeTime"); // load on demand
+const weekOfYear = require("dayjs/plugin/weekOfYear"); // load on demand
+const IsBetween = require("dayjs/plugin/IsBetween"); // load on demand
+const IsLeapYear = require("dayjs/plugin/IsLeapYear"); // load on demand
+dayjs.extend(RelativeTime);
+dayjs.extend(weekOfYear);
+dayjs.extend(IsBetween);
+dayjs.extend(IsLeapYear);
 
 const time = 1536484369695;
 
 describe("Parse", () => {
   it("String + Date Format", () => {
     const m = moment("12-25-1995", "MM-DD-YYYY");
     const d = date.parse("12-25-1995", "MM-dd-yyyy", new Date());
+    const day = dayjs("12-25-1995");
```

Comment:
make some naming convention here? like `date_fns_day` and `dayjs_day` to make it easier for other libs